docs/user-guide/content-sources.rst

```diff
@@ -98,6 +103,7 @@ The pulp-admin client can be use to list all defined content sources as follows:
   Base URL:       http://content-world/content/
   Enabled:        1
   Expires:        3d
+  Headers:        X-RHUI-ID=1234 \ X-Csrf-Token=31415926535897
```

Comment:
If on the same line, pulp-admin should do either:
1. strip the '\'
2. show on separate lines.

If not too much trouble, I would appreciate your fixing this.  Not a blocker for merge.